scss/sidebar.scss

```diff
@@ -0,0 +1,42 @@
+.header-sidebar-toggle {
+  button.widget-button {
+    border-radius: 0.5rem;
+    transition: border-radius 0.2s;
+    .d-icon {
+      color: var(--header_primary);
+      font-size: 1em;
+    }
+    &:hover {
+      border-radius: 50%;
+      background-color: var(--tertiary);
+      .d-icon {
+        color: var(--light-accent);
+      }
+    }
+  }
+}
+
+.sidebar-wrapper {
+  background: transparent;
+
+  .sidebar-section-header-text {
+    font-size: 0.75rem;
```

Comment:
we generally prefer `var(--font....)`

Reply:
Yeah I'm keeping consistent with other declarations in the theme for this, and perhaps can update all declarations at once to variables in a separate PR..